justfile

```diff
@@ -1,6 +1,11 @@
+# use powershell on windows
+set windows-shell := ["powershell.exe", "-NoLogo", "-Command"]
+
+# list available recipes by default
 _default:
   @just --list
 
+# build all flavors and example config files
```

Comment:
Are these comments needed? Seems pretty self explanatory to me but I guess.

Reply:
Wouldn't it povide explanation of each function at a glance without needing to go through the Just documentation?

Reply:
Maybe its just me but I see "default" next to "list" and I can tell that the default tasks will list available tasks. And the "build" task pretty clearly builds the themes :)

Reply:
Made a new commit removing the comments since the syntax is very readable. Thank you for your work!